record_a.go

```diff
@@ -0,0 +1,150 @@
+package ibclient
+
+import (
+	"fmt"
+	_ "time"
+)
+
+type RecordAOperations interface {
+	CreateARecord(recA RecordA) (*RecordA, error)
+	GetARecord(recA RecordA) (*[]RecordA, error)
+	DeleteARecord(recA RecordA) (string, error)
+	UpdateARecord(recA RecordA) (*RecordA, error)
+}
+type Dhcpmember struct {
+	Ipv4addr string    `json:"ipv4addr,omitempty"`
+	Ipv6addr string    `json:"ipv6addr,omitempty"`
+	Name     string    `json:"name,omitempty"`
+}
+type Cloud_info struct {
+	Authority_type      string       `json:"authority_type,omitempty"`
```

Comment:
please use camelcase instead of snake case

Reply:
> please use camelcase instead of snake case

Okay, I will change that.
Thank you for the feedback!